src/content/plugins/split-chunks-plugin.md

```diff
@@ -0,0 +1,230 @@
+
+It can match the absolute module resource path or chunk names. When a chunk name is matched, all modules in this chunk are selected.
+
+### Select chunks
+
+With the `chunks` option the selected chunks can be configured.
+
+There are 3 values possible `"initial"`, `"async"` and `"all"`. When configured the optimization only selects initial chunks, on-demand chunks or all chunks.
+
+The option `reuseExistingChunk` allows to reuse existing chunks instead of creating a new one when modules match exactly.
+
+This can be controlled per cache group.
+
+### `optimization.splitChunks.chunks: all`
+
+As it was mentioned before this plugin will affect dynamic imported modules. Setting the `optimization.splitChunks.chunks` option to `"all"` initial chunks will get affected by it (even the ones not imported dynamically). This way chunks can even be shared between entrypoints and on-demand loading.
+
+This is the recommended configuration.
+
+T> You can combine this configuration with the [HtmlWebpackPlugin](/plugins/html-webpack-plugin/), it will inject all the generated vendor chunks for you.
+
+## `optimization.splitChunks`
+
+This configuration object represents the default behavior of the `SplitChunksPlugin`.
+
+```js
+splitChunks: {
+	chunks: "async",
+	minSize: 30000,
+	minChunks: 1,
+	maxAsyncRequests: 5,
+	maxInitialRequests: 3,
+	name: true,
+	cacheGroups: {
+		default: {
+			minChunks: 2,
+			priority: -20,
+			reuseExistingChunk: true
+		},
+		vendors: {
+			test: /[\\/]node_modules[\\/]/,
+			priority: -10
+		}
+	}
+}
+```
+
+By default cache groups inherit options from `splitChunks.*`, but `test`, `priority` and `reuseExistingChunk` can only be configured on cache group level.
+
+`cacheGroups` is an object where keys are the cache group names. All options from the ones listed above are possible: `chunks`, `minSize`, `minChunks`, `maxAsyncRequests`, `maxInitialRequests`, `name`.
+
+You can set `optimization.splitChunks.cacheGroups.default` to `false` to disable the default cache group, same for `vendors` cache group.
+
+The priority of the default groups are negative to allow any custom cache group to take higher priority (the default value is `0`).
+
+Here are some examples and their effect:
+
+### Split Chunks: Example 1
+
+Create a `commons` chunk, which includes all code shared between entrypoints.
+
+```js
+splitChunks: {
+	cacheGroups: {
+		commons: {
+			name: "commons",
+			chunks: "initial",
+			minChunks: 2
+		}
+	}
+}
+```
+
+W> This downloads more code than neccessary.
+
+### Split Chunks: Example 2
+
+Create a `vendors` chunk, which includes all code from node_modules in the whole application.
+
+``` js
+splitChunks: {
+	cacheGroups: {
+		commons: {
+			test: /[\\/]node_modules[\\/],
+			name: "vendors",
+			chunks: "all"
+		}
+	}
+}
+```
+
+W> This downloads more code than neccessary.
```

Comment:
s/neccessary/necessary

Reply:
Also maybe elaborate this warning a bit...

> This will probably yield an unnecessarily large bundle as many applications use a bunch of `node_modules`. It's typically better to extract and load your core frameworks on initial load, and let other dependencies be chunked out and loaded dynamically later on.